packages/parser/src/transformations/categorical/charts/choropleth-transformation.ts

```diff
@@ -0,0 +1,48 @@
+import { GroupByResult, ParserSettings } from '../../../types';
+
+import { KEEN_KEY, KEEN_VALUE, KEEN_ELEMENTS } from '../../../constants';
+
+/**
+ * Choropleth chart transformation used for multiple group by properties
+ *
+ * @param result - categorical results
+ * @param parserSettings - parser settings
+ * @return transformed results
+ *
+ */
+export const choroplethChartTransformation = (
+  result: GroupByResult[],
+  _parserSettings: ParserSettings,
```

Comment:
it looks like it's not used anywhere

Reply:
Yes, this is why it have `_` prefix as it is a second function argument and all transformations accepts the unified arguments.

Reply:
👍